code/__HELPERS/turfs.dm

```diff
@@ -330,12 +330,15 @@ Turf and target are separate in case you want to teleport some distance from a t
 
 	return turf_list
 
+/* VOIDCREW EDIT - START
 ///Returns a random turf on the station
 /proc/get_random_station_turf()
 	var/list/turfs = get_area_turfs(pick(GLOB.the_station_areas))
```

Comment:
Maybe shuttle areas should just be part of ``GLOB.the_station_areas``, would be way more simple no?